src/main/java/frc/robot/subsystems/Gripper.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Gripper extends SubsystemBase{
```

Comment:
Are we using this at all? If not, we should delete this file.

Reply:
Gripper Subsystem and Command file go bye bye!